### ISSUE TITLE
[a11y] test

### ISSUE THREAD

**Issue by monicawheeler** [ju-Skinner/sturdy-dollop#3]
## Environment

- OS: Mac
- Browser: Firefox
- Testing tool used: VO on Mac
- Assistive technology used to verify: VO on Mac

## Detailed description

- What did you expect to happen? Navigation to read in order
- What happened instead? Navigation read out of order
- What WCAG 2.1 success criteria does the issue violate? 2.4.3 Focus Order

## Steps to reproduce the issue

2. Click on '...'
3. Scroll down to '...

## Additional Information

- Screenshots or code
- Notes
